diff --git a/out/fp0047-baseline/web_string.zig b/src/web_string.zig
index 124be09..737c047 100644
--- a/out/fp0047-baseline/web_string.zig
+++ b/src/web_string.zig
@@ -261,7 +261,7 @@ const Utf8Decoder = struct {
                         bytes_needed = 2;
                         code_point = byte & 0x0F;
                     },
-                    0xF0...0xF4 => {
+                    0xF0...0xF7 => {
                         if (byte == 0xF0) lower_boundary = 0x90;
                         if (byte == 0xF4) upper_boundary = 0x8F;
                         bytes_needed = 3;
