diff --git a/src/unicode/tables.zig b/src/unicode/tables.zig
index 8d214b7..9558f6c 100644
--- a/src/unicode/tables.zig
+++ b/src/unicode/tables.zig
@@ -11307,7 +11307,7 @@ pub const grapheme_cluster_break_ranges = [_]Range(GraphemeClusterBreak){
     .{ .start = 0x0008CA, .value = .EX },
     .{ .start = 0x0008E2, .value = .PP },
     .{ .start = 0x0008E3, .value = .EX },
-    .{ .start = 0x000903, .value = .SM },
+    .{ .start = 0x000903, .value = .XX },
     .{ .start = 0x000904, .value = .XX },
     .{ .start = 0x00093A, .value = .EX },
     .{ .start = 0x00093B, .value = .SM },
